acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/multiplatformsigner/AzureSignerMultiplatformSigning.java

```diff
@@ -0,0 +1,161 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+public class AzureSignerMultiplatformSigning {
+
+  private static final String clientId = System.getenv("ETHSIGNER_AZURE_CLIENT_ID");
+  private static final String clientSecret = System.getenv("ETHSIGNER_AZURE_CLIENT_SECRET");
+
+  private final String AZURE_ETHEREUM_ADDRESS = "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73";
+  private final String FILE_ETHEREUM_ADDRESS = "0xa01f618424b0113a9cebdc6cb66ca5b48e9120c5";
+  private Signer ethSigner;
+
+  @TempDir Path tomlDirectory;
+
+  @BeforeAll
+  public static void preChecks() {
+    Assumptions.assumeTrue(
+        clientId != null && clientSecret != null,
+        "Ensure Azure client id and client secret env variables are set");
+  }
+
+  @AfterEach
+  public void cleanUp() {
+    if (ethSigner != null) {
+      ethSigner.shutdown();
+      ethSigner = null;
+    }
+  }
+
+  public void setup() {
+    final SignerConfiguration signerConfig =
+        new SignerConfigurationBuilder().withMultiplatformSignerDirectory(tomlDirectory).build();
+    final NodeConfiguration nodeConfig = new NodeConfigurationBuilder().build();
+
+    ethSigner = new Signer(signerConfig, nodeConfig, new NodePorts(1, 2));
+    ethSigner.start();
+    ethSigner.awaitStartupCompletion();
+  }
+
+  private void createAzureTomlFileAt(final String tomlFilename) {
+    try {
+      final Path tomlFilePath = tomlDirectory.resolve(tomlFilename);
+
+      final FileWriter writer = new FileWriter(tomlFilePath.toFile(), StandardCharsets.UTF_8);
+      writer.write("[signing]\n");
+      writer
+          .append("type = \"azure-based-signer\"\n")
+          .append("key-vault-name = \"ethsignertestkey\"\n")
+          .append("key-name = \"TestKey\"\n")
+          .append("key-version = \"7c01fe58d68148bba5824ce418241092\"\n")
+          .append("client-id = \"" + clientId + "\"\n")
+          .append("client-secret = \"" + clientSecret + "\"\n");
+      writer.close();
+    } catch (final IOException e) {
+      fail("Unable to create Azure TOML file.");
+    }
+  }
+
+  private void createFileBasedTomlFileAt(
+      final String tomlFilename, final String keyPath, final String passwordPath) {
+    try {
+      final Path tomlFilePath = tomlDirectory.resolve(tomlFilename);
+
+      final FileWriter writer = new FileWriter(tomlFilePath.toFile(), StandardCharsets.UTF_8);
+      writer.write("[signing]\n");
+      writer
+          .append("type = \"file-based-signer\"\n")
+          .append("key-file = \"" + keyPath + "\"\n")
+          .append("password-file = \"" + passwordPath + "\"\n");
+      writer.close();
+    } catch (final IOException e) {
+      fail("Unable to create Azure TOML file.");
+    }
+  }
+
+  @Test
+  void multipleSignersAreCreatedAndExpectedAddressAreReported() throws URISyntaxException {
```

Comment:
This test isn't specific to Azure so should be in a general multi-platform  AT I think